review/reviewer/looking-for.md

```diff
@@ -179,6 +179,27 @@ mistakes, but they should offer encouragement and appreciation for good
 practices, as well. It’s sometimes even more valuable, in terms of mentoring, to
 tell a developer what they did right than to tell them what they did wrong.
 
+## Feature Flags
+
+If you notice that a new feature flag is being introduced you should verify that
+the feature flag key also exists in the feature flag service (LaunchDarkly). If
+it does not, then inform the code author that it needs to be addressed before
+production deployment, ideally right away. If the application has a
+`.launch-darkly/feature-flags.json` file, then it should also contain the new
+key.
+
+If the last occurrence of a feature flag is being removed, then it should be
+removed from the feature flag service (LaunchDarkly) and the
```

Comment:
> I don't think we should mention removing it from LaunchDarkly here in the code review. This is because we don't want this flag removed until the code has been merged and deployed.

I think this is referring to the process where we remove feature flags, which obv comes later, but should be considered for the process

Reply:
sorry, ignore my comment, I get what you mean

Reply:
sorry, ignore my comment, I get what you mean